recipe/meta.yaml

```diff
@@ -26,30 +26,26 @@ requirements:
   build:
   - cmake>=3.10
   - eigen
-#  - eigen=3.3.3
   - gsl
-#  - gsl==2.2.1
+  - netowrkx
   - numpy
   - openblas
-#  - openblas==0.2.19
   - python
-#  - python==3.6
   - python-dateutil
   - pvl
+  - pysis
```

Comment:
Most of these are likely not build dependencies. I believe only cmake, eigin, and gsl are build dependencies. Then python is a host dependency. Everything else should be run time dependencies.

Also, is ISIS/pysis an ALE dependency?

Reply:
We currently have pysis under the utils module. It probably needs to be taken out.